doc/api/http2.md

```diff
@@ -1645,6 +1657,18 @@ negotiate an allowed protocol (i.e. HTTP/2 or HTTP/1.1). The event handler
 receives the socket for handling. If no listener is registered for this event,
 the connection is terminated. See the [Compatibility API][].
 
+#### server.close([callback])
+<!-- YAML
+added: v0.3.2
```

Comment:
Can you make sure that these versions are right? I think it’s 8.4.0, the version that the whole `http2` module was originally introduced in